elements/lisk-tree/src/sparse_merkle_tree/types.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+export interface KeyValueStore {
```

Comment:
```suggestion
export interface Database {
```
I added the interface as `Database` on standard merkle tree, so let's keep it the same